### DISCUSSION TITLE
Support for payments using debit cards?

### DISCUSSION THREAD

**Discussion by djc** [community/community#45195] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Like some others who have started discussion in https://github.com/orgs/community/discussions/45183 and https://github.com/orgs/community/discussions/45180, I was not happy to see that Paypal support is being discontinued. However, it seemed okay because the emails I got mentioned to "please update your payment method for GitHub to pay by credit or debit card." As a Dutch user, credit cards are not a very common payment method here (and they're expensive relative to the alternatives), but paying by debit card would be a pretty good option. However, upon arriving on https://github.com/settings/billing/payment the only options I see are "Credit card" and "Paypal account". What gives?

**Comment by Phoenix616** [comment]
Aren't debit cards behaving essentially like credit cards in most cases? Has anyone tested that this isn't the case here?

Besides that: Allowing debit doesn't even solve this. Lots of banks don't even offer that. Supporting SEPA is a must to reach people in the EU and PayPal allowed that!

**Reply by adiati98** [reply]
I was about to open a discussion and saw this thread. 

I'm also based in The Netherlands, and the reason I'm using PayPal as payment method is that because I don't have any credit card myself.

PayPal allows me to use my debit card. I wish there's another option as PayPal to allow payment from Dutch debit card. Otherwise, I don't have any other choice than to stop sponsoring my communities 😔

Hopefully, someone from GitHub can answer us.

**Reply by maxiride** [reply]
> Aren't debit cards behaving essentially like credit cards in most cases? Has anyone tested that this isn't the case here?

The main issue is the number of digits of the card. Maestro and Visa Electron have up to 19 digits and the input field on the Github billing page doesn't allow you to enter more than 16.

**Reply by Nazo1432** [reply]
Not allowed

**Comment by lucacasonato** [comment]
I second this. If GH does not support PayPal anymore, at least offer SEPA - most individuals in the Netherlands do not have a credit card. We pay using Maestro/VPay direct debit cards, which are not the same as MasterCard/Visa debit cards. Online we pay with PayPal / iDEAL / SEPA. You can not use Maestro/VPay cards online.

If you had done this a year later, it may have made sense, because Mastercard is phasing out Maestro in Europe, but that transition won't be complete for a long time. For now many people in many EU countries do not have credit cards.

Please support SEPA direct debit - the payment processing fees that Stripe charges for SEPA are lower than for credit cards (they are fixed €0.35), and they support recurring payments just like PayPal and credit card: https://stripe.com/docs/payments/sepa-debit.

**Reply by Nazo1432** [reply]
Not allowed

**Comment by siccovansas** [comment]
Debit cards are now available at certain banks in the Netherlands. When I tried to add mine to GitHub I got an 'The credit card was declined by the payment processor' error. I fixed this by configuring my card to allow world/international transactions.

**Reply by djc** [reply]
I'm in NL as well, but when adding a debit card you have to enter a card number that looks like nothing like the card numbers on my usual checking account debit card. How did you get this to work? Are you actually using a credit card?

**Reply by siccovansas** [reply]
Debit cards are [being rolled out in the Netherlands](https://www.maxvandaag.nl/sessies/themas/geld-werk-recht/van-maestro-en-v-pay-naar-debit-mastercard-en-visa-debit-wat-moet-u-weten-over-de-nieuwe-betaalpassen/). Some banks already have them, others will the coming months(/years?). If your bank card doesn't have a credit card-like number then it is not a debit card (yet).

**Comment by cyrneko** [comment]
Still annoyed by this, my German Debit card's info won't even fit... -_-

**Reply by kissmycodee** [reply]
What did you do to solve ?

**Reply by borislineup7** [reply]
I added my card to paypal and set it as the payment method for paypal, then chose my card to pay the bills. it works.

**Reply by kissmycodee** [reply]
Your domestic bank card with IBAN or something ? In paypal i did not add a credit or debit card ?

**Reply by borislineup7** [reply]
https://www.paypal.com/us/cshelp/article/how-do-i-link-a-debit-or-credit-card-to-my-paypal-account-help155

**Reply by kissmycodee** [reply]
But i dont have debit or credit card just my normal bank card or account is linked to paypal ?

**Comment by glozanomoran** [comment]
My apologies for reviving this discussion. I am Belgian-based, I have setup a new organization and I am trying to setup billing using a debit card that uses MASTERCARD instead of MAESTRO. I am getting the following error after getting a HTTP 500 error:
'Invalid payment method - authorization hold failed.' 

And now when I try again, I am getting the following error:
'The credit card was declined by the payment processor, please verify the card's information. If the problem persists, please contact the card's issuer.'

The funny thing is that the exact same debit card was working perfectly fine with my personal account.

I have also tried a MASTERCARD credit card that allows worldwide payments, and I get the same error:
'Invalid payment method - authorization hold failed.' 

What is the issue here? I have already created a support ticket but I was kind of hoping that anybody else here had the same issue and found a solution or a work-around.

**Reply by borislineup7** [reply]
Hello,

I am in France and we have a new organization for our company.

I have exactly the same problem and have had no response from support for almost 24 hours. 

@glozanomoran Have you solved your problem? Support response? 

**My context:** 

Our company has several credit cards that did not go through when we added them. We had to use our CEO's personal card.

At that time, I bought the “Team” plan with 8 seats for 1 year.

This happened in September.

My CEO wanted us to withdraw his bank card and we withdrew it from the organization.

At the beginning of the month, we needed 2 additional seats and activation of Copilot for 7 members, so yesterday, October 9, we gave his personal bank card back because the company's was still blocked.... we added but there was no security request.

I added 2 seats for 1 year in the team plan + 7 copilot seats and a few hours later we got the following message **“Invalid payment method - authorization hold failed.”**

Since then, the entire organization has been blocked from the “team” plan options:
- **“Failed to create the environment protection rule. Please ensure the billing plan supports the wait timer protection rule.”**

- **"Upgrade to GitHub Pro or make this repository public to enable this feature"**

We have tried to add another card, a paypal account, but at the moment of validation, we have an error page 500.

We also have the following banner: **"We are having a problem billing the **** organization. Please [update your payment information](https://github.com/organizations/*****/settings/billing/summary) or call your payment provider for details on why the transaction failed."**

We find ourselves blocked, without being able to add a means of payment and with our organization blocked even though we paid the invoice in September for the team plan for 8 seats.

Thanks in advance for your reply!

Have a nice day!

**Reply by glozanomoran** [reply]
The funny thing is that support did contact me and had to reset the GitHub Copilot licenses. I was then able to assign and purchase the seats by going through the wizard from scratch. When I look at the subscription details, I don't see any payment details, and there was no payment withheld from my Mastercard debit card although my GitHub Copilot license is fully functional. I have no idea what's going on. It seems like something went wrong, and now AFAIK, I have a free seat for GitHub Copilot.

**Reply by borislineup7** [reply]
Great! That's really good for you for the free licenses and it's the least you can do given the concern and in compensation for the problem.

It would be the least I could do in my case.

This assumes that the GitHub Copilot licenses are the source of the problem or linked to it. 

I'll wait for support to get back to me and thanks for your quick reply!

**Reply by glozanomoran** [reply]
I’m not 100% sure my license is okay because earlier today I hit a token limit in the Copilot chat window in Visual Studio, which I don’t recall seeing before. So, it seems like this might be a free trial. 

When I switch to my glozanomoran account, it shows 'GitHub Copilot Business is active for your account' and 'You are assigned a seat as part of a GitHub Copilot Business subscription managed by MONOTECHNICS'.

When I switch to my MONOTECHNICS context I see 'Copilot Business is active in your organization' but when I click on 'Review billing details' I see that the current monthly bill is '$0', there is no 'Next payment due', the list in 'View payment history' is empty, etc. And I do not see anywhere where I can change my payment details.

**Comment by CJAC-TDI-ADM** [comment]
Hi,
Check your spending Limit. The default value is $0.
I had the same problem. The GitHub support team also pointed me towards an issue with my bank or my credit card.
Hoping this helps future users.

**Comment by alienarena** [comment]
Can you please add support for payments with other payment methods than PayPal or Credit cards, like a SEPA direct debit or Ideal? In my case not for GitHub Sponsors but for Github Copilot. In the Netherlands most people don't have a credit card (or debit card in the sense that is needed here).

**Comment by marcinpraczko** [comment]
I have very similar situation like:
- https://github.com/orgs/community/discussions/45195#discussioncomment-10808792

I am using PREPAID card and now I've got this message:
- The credit card was declined by the payment processor, please verify the card's information. If the problem persists, please contact the card's issuer.

And I don't have any credit card and I don't want provide debit card. This is a reason that I am adding pre-paid as I have my budget which I am controlling.
Can GitHub explain me why this is deny, please. I would like to understand reasons why pre-paid cards which are so well know and used are not allowed.
